### ISSUE TITLE
Pattern [0-9,\s]{5} generates whitespace symbols that user probably didn't expect - make configurable?

### ISSUE THREAD

**Issue by mockmotor** [curious-odd-man/RgxGen#77]
A simple pattern [0-9,\s]{5} generates strings like (in hex):

0035 0031 0020 **000d** **000c** 
**0009** 0035 **000c** 0035 0035 

I.e. using CR and LF and also symbols such as FF (form feed) and TAB.

Formally, it is correct - those control characters match the \s format.

However, in practice, users only think of spaces and maybe tabs when using the \s format.

It would be convenient to make this behaviour configurable via options.

I propose three options:

* \s expands only into space character
* \s expands into space and tab
* \s expands into any symbol in Character.isWhitespace() 

In my code, I manually replace the whitespace characters with spaces. It is a workaround, but a bit ugly :)

**Comment by curious-odd-man** [comment]
Hello! Just out of curiosity - if you say that this pattern gives you unexpected characters, why don't you use a pattern that would produce only expected values? 
I mean instead of `\s` you could use space and tab characters - then only those characters would be used.

**Comment by mockmotor** [comment]
Hey there.

The schemas (WSDLs) I receive are not mine. Those are from external teams. My goal is to generate reasonable mock payloads for the requests and responses - I maintain a mock service, https://mockmotor.com .

So when the external schema contains types like

  <simpleType name="SpinID">
    <restriction base="string">
      <pattern value="[0-9,\s]{14}" />
    </restriction>
  </simpleType>

They don't expect to see new lines in the generated values, let alone the tabs and LF.

In fact, when a value can include a multiline value or a general text, they just use type xs:string. 

The very fact of using a pattern signals to me that the value is very likely a single-line value.

**Comment by curious-odd-man** [comment]
Hello! Thank you for the explanation. I'll implement the requested change.

**Comment by curious-odd-man** [comment]
Hello @mockmotor !

This is now implemented as part of 2.0 release.
2.0 version will be release in future, for now - you can test/use this feature from the 2.0-SNAPSHOT version

```
<project>
    <repositories>
        <repository>
            <id>snapshots-repository</id>
            <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
        </repository>
    </repositories>

    <!--  .... -->

    <dependency>
        <groupId>com.github.curious-odd-man</groupId>
        <artifactId>rgxgen</artifactId>
        <version>2.0-SNAPSHOT</version>
    </dependency>
</project>
```

**Comment by mockmotor** [comment]
Excellent, thanks! I will test it for the next week's build!
